scripts/SANS/sans/algorithm_detail/move_workspaces.py

```diff
@@ -451,17 +464,19 @@ def _move_monitor_4(workspace, move_info):
             detector_position = lab_detector_component.getPos()
             z_position_detector = detector_position.getZ()
 
-            monitor_4_offset = move_info.monitor_4_offset / 1000.
+            monitor_4_offset = move_info.monitor_4_offset
```

Comment:
Yes, we convert mm to m already in the state builder, hence this was a mistake to convert it again.